index.html

```diff
@@ -1981,9 +1981,9 @@ <h3>Combining Value Constraints</h3>
           </div>
             <table class="example results">
               <tbody><tr><th class="data">Node</th><th class="schema">Shape</th><th>Result</th><th>Reason</th></tr>
-              <tr class="pass"><td>inst:User1</td><td>my:UserShape</td><td>pass</td><td class="noreason"></td></tr>
+              <tr class="pass"><td>inst:Issue1</td><td>my:IssueShape</td><td>pass</td><td class="noreason"></td></tr>
               <tr class="pass"><td>&lt;http://hr.example/id#123&gt;</td><td>my:EmployeeShape</td><td>pass</td><td class="noreason"></td></tr>
-              <tr class="fail hilight-component top lowlight2"><td>inst:User3</td><td>my:UserShape</td><td>fail</td><td class="fail">Object of <span class="predicate"><span class="type">ex:</span><span class="constant">reproducedBy</span></span> does not match <span class="shape-name">my:EmployeeShape</span>.</td></tr>
+              <tr class="fail hilight-component top lowlight2"><td>inst:Issue1</td><td>my:IssueShape</td><td>fail</td><td class="fail">Object of <span class="predicate"><span class="type">ex:</span><span class="constant">reproducedBy</span></span> does not match <span class="shape-name">my:EmployeeShape</span>.</td></tr>
```

Comment:
We agree.

Reply:
Accept his change, then after accepting:
* second inst:Issue1 => inst:Issue3
* change Shape Map
* change try-it links